reference/v8.json

```diff
@@ -607,6 +608,11 @@
           "js": "0.10.0",
           "ios": "2.0.0",
           "android": "2.0.1"
+        },
+        "auto": {
+          "js": "TBD",
```

Comment:
> let me know when we should add the specific version (at least on the GL JS side)

Our policy is to update the style spec once the feature goes out in a public release. The release coordinator is responsible for this update.